From 967a11d45488a1776323687b6164bac16de5b7d4 Mon Sep 17 00:00:00 2001
From: Evan Quan <evan.quan@amd.com>
Date: Mon, 5 Jun 2017 17:15:02 +0800
Subject: [PATCH 2178/2500] drm/amdgpu: coordinate with the new
 amdgpu_queue_mgr_map - 2c5e7b4 drm/amdgpu: untie user ring ids from kernel
 ring ids v6

Change-Id: I084d47eb007e88764d7b047f08e7719e94dc47d8
Signed-off-by: Evan Quan <evan.quan@amd.com>
Reviewed-by: Junwei Zhang <Jerry.Zhang@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_sem.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_sem.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_sem.c
index 2bea5f6..7852c8b 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_sem.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_sem.c
@@ -338,8 +338,8 @@ static struct fence *amdgpu_sem_get_fence(struct amdgpu_fpriv *fpriv,
 	ctx = amdgpu_ctx_get(fpriv, ctx_id);
 	if (!ctx)
 		return NULL;
-	r = amdgpu_cs_get_ring(ctx->adev, ip_type, ip_instance, ring,
-			       &out_ring);
+	r = amdgpu_queue_mgr_map(ctx->adev, &ctx->queue_mgr, ip_type,
+			       ip_instance, ring, &out_ring);
 	if (r) {
 		amdgpu_ctx_put(ctx);
 		return NULL;
@@ -368,8 +368,8 @@ static int amdgpu_sem_cring_add(struct amdgpu_fpriv *fpriv,
 	ctx = amdgpu_ctx_get(fpriv, ctx_id);
 	if (!ctx)
 		return -EINVAL;
-	r = amdgpu_cs_get_ring(ctx->adev, ip_type, ip_instance, ring,
-			       &out_ring);
+	r = amdgpu_queue_mgr_map(ctx->adev, &ctx->queue_mgr, ip_type,
+			         ip_instance, ring, &out_ring);
 	if (r)
 		goto err;
 
-- 
2.7.4

